newrelic/resource_newrelic_application_test.go

```diff
@@ -0,0 +1,71 @@
+package newrelic
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/terraform"
+)
+
+func TestAccNewRelicApplication_Basic(t *testing.T) {
+	resourceName := "newrelic_application.app"
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckNewRelicApplicationDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccNewRelicApplicationConfig(),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckNewRelicApplicationExists(testAccExpectedApplicationName),
+					resource.TestCheckResourceAttr(
```

Comment:
Fairly certain the **import** test handles verifying these values so no need to include them.

cc: @ctrombley

Reply:
We can still keep `testAccCheckNewRelicApplicationExists` though. That's a more custom check.

Reply:
Have done.